### PR TITLE
Plasmid finder

### PR DESCRIPTION
**Pull request by pimarin**

### CODE REVIEW

**Comment by pimarin** [comment]
The tests failed because :
`Job in error state.. tool_id: plasmidfinder, exit_code: 127, stderr: /bin/bash: /tmp/tmpjaxtnhr0/job_working_directory/000/2/tool_script.sh: No such file or directory`
But only whith container and I don't understand why

**Comment by pimarin** [comment]
Any idea ?

File: tools/plasmidfinder/plasmidfinder.xml [hunk 1]
```diff
@@ -31,24 +31,24 @@
...
         </section>
         <section name="options" title="Options">
-            <param name="min_cov" type="float" min="0" max="1" value="0.6" label="Minimal coverage" help="Choose a minimum coverage value (default: 0.6)"/>
-            <param name="threshold" type="float" min="0" max="1" value="0.95" label="Minimal identity" help="Choose a minimum identity value (default: 0.95)"/>
+            <param name="min_cov" type="float" min="0" max="1" value="0.6" label="Minimal coverage" help=" % of matching sequence on the total target sequence(default: 0.6)"/>
+            <param name="threshold" type="float" min="0" max="1" value="0.95" label="Minimal identity" help=" % of shared nucleotide on the match (default: 0.95)"/>
         </section>
         <section name="output_files" title="Output file selection">
-            <param name="output_selection" type="select" display="checkboxes" multiple="true" label="Output files selection">
+            <param name="output_selection" type="select" display="checkboxes" multiple="true" optional="true" label="Output files selection">
```
Comment (bernt-matthias) [inline tools/plasmidfinder/plasmidfinder.xml:46]: I would not make this optional. Otherwise it might happen that the tool has no outputs.

**Comment by pimarin** [comment]
Is it okay ?

**Review by bgruening — APPROVED** [review]

**Comment by bgruening** [comment]
Thanks a lot @pimarin!

**Comment by pimarin** [comment]
Hum it's strange this file is in the tool-data and plasmidfinder_db.loc is also in test-data

**Comment by pimarin** [comment]
I pushed again with plasmidfinder_db.loc.sample and plasmidfinder_db.loc in tool-data and test-data
test on my computer are okay

**Pull request merged**